src/ISA/ISA/ArcTypes/IdentifierSetters.fs

```diff
@@ -2,6 +2,11 @@
 
 open ARCtrl.ISA.Identifier
 
+let setArcTableName (newName: string) (table: ArcTable) =
```

Comment:
Why not "ArcTable.setName"?

Reply:
This must be unique right? So we only allow updating this unsafe. ArcTables implements a rename option

Reply:
Ah you mean you wanted to hide the functionality?

Reply:
Exactly as the other unique identifier setters

Reply:
We have a visible member for this, but only on higher level types where we can sanitycheck the changes